src/group_theory/free_group.lean

```diff
@@ -560,7 +560,7 @@ prod.of
 instance sum.is_group_hom : is_group_hom (@sum α _) :=
 prod.is_group_hom
 
-@[simp] lemma sum.sum : sum (x * y) = sum x + sum y :=
+@[simp, nolint] lemma sum.sum : sum (x * y) = sum x + sum y :=
```

Comment:
This should probably be renamed `sum.mul`, but I didn't want to get into semantic changes in this PR.

Reply:
Particularly as `nolint` disables *all* checks, I think it would be good to try to have a brief comment explaining the purpose of each `nolint`. We can add them in a later PR, though.

Reply:
Ah, now I see Johan wrote pretty much the same thing.